Bugzilla/Elastic/Indexer.pm

```diff
@@ -238,7 +181,6 @@ sub _bulk_load_ids {
     }
 
     my $total = 0;
-    use Time::HiRes;
```

Comment:
It shouldn't have been there to begin with -- but at the top level. Sorry for the unrelated change.